src/mmw/js/src/core/utils.js

```diff
@@ -73,6 +73,54 @@ var utils = {
             });
 
         return md5(JSON.stringify(sortedCollection));
+    },
+
+    convertToMetric: function(value, fromUnit) {
```

Comment:
Right. Good point. I'm surprised there aren't that many great libraries out that there that do this (I looked around a bit). I think it's fine as is.